.travis.yml

```diff
@@ -16,14 +16,14 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=4.0
+      env: SWIFT_SNAPSHOT=4.0.2
```

Comment:
Since PackageBuilder pulls from the swift-version the env variables can be removed.